WebRoot/messages/ZaMsg_hu.properties

```
@@ -1199,6 +1199,7 @@ NAD_ForwardTo = Tov\u00e1bb\u00edt\u00e1s adott c\u00edmre
 NAD_COSName = COS-n\u00e9v
 NAD_PwdLocked = Jelsz\u00f3m\u00f3dos\u00edt\u00e1s tilt\u00e1sa a felhaszn\u00e1l\u00f3 sz\u00e1m\u00e1ra
 NAD_AllowUsernameInPassword = Felhaszn\u00e1l\u00f3n\u00e9v enged\u00e9lyez\u00e9se a Jelsz\u00f3ban
+NAD_PasswordExpiryReminder = Jelszó lejárati emlékeztet\u0151
```

We need to convert latin letters to respective unicode.
```suggestion
NAD_PasswordExpiryReminder = Jelsz\u00f3 lej\u00e1rati eml\u00e9keztet\u0151
```

---

Done